compiler/src/dmd/globals.d

```diff
@@ -182,6 +182,7 @@ extern (C++) struct Param
     CHECKACTION checkAction = CHECKACTION.D; // action to take when bounds, asserts or switch defaults are violated
 
     uint errorLimit = 20;
+    uint errorSupplementLimit = 6;      // Limit the number of supplemental messages for each error (0 means unlimited)
```

Comment:
Why not maintaining the old hard coded limit, 5, instead of 6?

Reply:
It replaces three hardcoded numbers .  5, 6, and 6.  6 won.  ;)

Reply:
Ah, fair, didn't catch the other 6 :)